content/en/blog/_posts/2021-12-15-pod-security-admission-psa-beta.md

```diff
@@ -0,0 +1,620 @@
+  - name: nginx
+    image: nginx
+    securityContext:
+      allowPrivilegeEscalation: true
+EOF
+```
+
+The output is similar to this:
+
+```shell
+pod/nginx-privileged created
+```
+
+We can run `kubectl -n kube-test-privileged get pods` to verify it is running.
+
+**Baseline level and workload**
+
+The baseline pod demonstrates sensible defaults but does not allow privilege escalation, so the pod above would fail here too ([more info](https://kubernetes.io/docs/concepts/security/pod-security-standards/#baseline)). Baseline accepts null / no values for unset keys.
+
+First, let's try to deploy it in the restricted namespace:
+
+```shell
+cat <<EOF | kubectl -n kube-test-restricted apply -f -
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx-baseline
+spec:
+  containers:
+  - name: nginx
+    image: nginx
+    securityContext:
+      allowPrivilegeEscalation: false
+      capabilities:
+        add:
+          - NET_BIND_SERVICE
+          - CHOWN
+EOF
+```
+
+The output is similar to this:
+
+```shell
+Error from server (Forbidden): error when creating "STDIN": pods "nginx-baseline" is forbidden: violates PodSecurity "restricted:latest": unrestricted capabilities (container "nginx" must set securityContext.capabilities.drop=["ALL"]; container "nginx" must not include "CHOWN" in securityContext.capabilities.add), runAsNonRoot != true (pod or container "nginx" must set securityContext.runAsNonRoot=true), seccompProfile (pod or container "nginx" must set securityContext.seccompProfile.type to "RuntimeDefault" or "Localhost")
+```
+
+Now let's try the `kube-test-baseline` namespace
+
+```shell
+cat <<EOF | kubectl -n kube-test-baseline apply -f -
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx-baseline
+spec:
+  containers:
+  - name: nginx
+    image: nginx
+    securityContext:
+      allowPrivilegeEscalation: false
+      capabilities:
+        add:
+          - NET_BIND_SERVICE
+          - CHOWN
+EOF
+```
+
+The output is similar to this:
+
+```shell
+Warning: would violate PodSecurity "restricted:latest": unrestricted capabilities (container "nginx" must set securityContext.capabilities.drop=["ALL"]; container "nginx" must not include "CHOWN" in securityContext.capabilities.add), runAsNonRoot != true (pod or container "nginx" must set securityContext.runAsNonRoot=true), seccompProfile (pod or container "nginx" must set securityContext.seccompProfile.type to "RuntimeDefault" or "Localhost")
+pod/nginx-baseline created
+```
+
+Remember, we set the `kube-test-baseline` namespace to `warn` based on the restricted profile. We can run `kubectl -n kube-test-baseline get pods` to verify it is running.
+
+**Restricted level and workload**
+
+Restricted requires rejection of all privileged parameters. It is the most secure with a trade-off for complexity ([more info](https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted)). The restricted policy allows containers to add the `NET_BIND_SERVICE` capability.
```

Comment:
Same comment as earlier - I suggest that you make the word "Restricted" the link text, not "more info".